crates/bevy_ui/src/render/mod.rs

```diff
@@ -128,25 +124,36 @@ pub fn build_ui_render(app: &mut App) {
                 RenderUiSystem::ExtractText,
             )
                 .chain(),
-        )
-        .add_systems(
-            ExtractSchedule,
-            (
-                extract_default_ui_camera_view,
-                extract_uinode_background_colors.in_set(RenderUiSystem::ExtractBackgrounds),
-                extract_uinode_images.in_set(RenderUiSystem::ExtractImages),
-                extract_uinode_borders.in_set(RenderUiSystem::ExtractBorders),
-                extract_text_sections.in_set(RenderUiSystem::ExtractText),
-            ),
-        )
-        .add_systems(
-            Render,
-            (
-                queue_uinodes.in_set(RenderSet::Queue),
-                sort_phase_system::<TransparentUi>.in_set(RenderSet::PhaseSort),
-                prepare_uinodes.in_set(RenderSet::PrepareBindGroups),
-            ),
         );
+    #[cfg(feature = "bevy_text")]
```

Comment:
There's too much duplication here. Add the systems unconditionally, and then just conditionally add extract_text_sections.